src/neo/Network/P2P/Peer.cs

```diff
@@ -208,27 +211,23 @@ private void OnStart(ChannelsConfig config)
         private void OnTcpConnected(IPEndPoint remote, IPEndPoint local)
```

Comment:
I think we can mark this method as `virtual`, and move the `PreTcpConnectedCheck()` logic to the overriding method.

Reply:
It's good for me, as it seems to be the design at first. I'll do it.

Reply:
Done